src/Data/List/Membership/Setoid/Properties.agda

```diff
@@ -122,6 +122,15 @@ module _ (S : Setoid c ℓ) where
                     length (mapWith∈ xs f) ≡ length xs
   length-mapWith∈ []       = P.refl
   length-mapWith∈ (x ∷ xs) = P.cong suc (length-mapWith∈ xs)
+  
+  mapWith∈-id : ∀ xs → mapWith∈ xs (λ {x} _ → x) ≡ xs
+  mapWith∈-id [] = P.refl
+  mapWith∈-id (x ∷ xs) rewrite mapWith∈-id xs = P.refl
+
+  map∘mapWith∈ : ∀ {a b} {A : Set a} {B : Set b} xs (f : ∀ {x} → x ∈ xs → A) (g : A → B) →
```

Comment:
Very minor but could we break this line at `∀ {a b} {A : Set a} {B : Set b}→` and `∀ xs (f : ∀ {x} → x ∈ xs → A) (g : A → B)`?